oap-server/server-receiver-plugin/skywalking-browser-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/recevier/browser/provider/parser/errorlog/listener/errorlog/SourceBuilder.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.oap.server.recevier.browser.provider.parser.errorlog.listener.errorlog;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.apache.skywalking.oap.server.core.browser.source.BrowserAppErrorLog;
+import org.apache.skywalking.oap.server.core.browser.source.BrowserAppPageErrorLog;
+import org.apache.skywalking.oap.server.core.browser.source.BrowserErrorCategory;
+
+/**
+ * @author zhangwei
+ */
+@Setter
+@Getter
+class SourceBuilder {
```

Comment:
There is already a `SourceBuilder`, please don't use duplicated name.  Also, the existing one should be renamed to `Trace SourceBuilder`.